internal/resources/standalone/index-template.html

```diff
@@ -97,6 +97,16 @@
       padding: 24px;
     }
     </style>
+    <script>
+    // ZOMG this is awful. This beast comes from https://developer.mozilla.org/en-US/docs/Web/API/document/cookie
```

Comment:
Yeah that is monstrous. It seems like if it's consistent enough to write a regex for that, then they could have made a decent programming API.